Fix json

SurvExe1Pc opened the pull request:
this will fix it

SurvExe1Pc commented:
related #16 also when this is merged wait for the site to be built and deployed

SurvExe1Pc commented:
@***@***.***> I message you on here cause I cant get on discord until 10/16/2023, hope you understand lol (im using email cause i cant log into github on my school computer).
________________________________
From: SharkPool-SP ***@***.***>
Sent: Wednesday, August 16, 2023 8:35 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


Merged #16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16> into main.

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#event-10113739368>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIBWMEFSUVEJUHY7CZ3XVVRMHANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SharkPool-SP commented:
> @***@***.***> I message you on here cause I cant get on discord until 10/16/2023, hope you understand lol (im using email cause i cant log into github on my school computer).
> […](#)
> ________________________________ From: SharkPool-SP ***@***.***> Sent: Wednesday, August 16, 2023 8:35 PM To: SharkPool-SP/SharkPools-Extensions ***@***.***> Cc: Surv ***@***.***>; Author ***@***.***> Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16) Merged #16<#16> into main. — Reply to this email directly, view it on GitHub<[#16 (comment)](https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#event-10113739368)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIBWMEFSUVEJUHY7CZ3XVVRMHANCNFSM6AAAAAA3SW6T4E>. You are receiving this because you authored the thread.Message ID: ***@***.***>

okay, thanks for letting me know :)

SurvExe1Pc commented:
if possible do you have email i could message you at?
________________________________
From: SharkPool-SP ***@***.***>
Sent: Tuesday, September 19, 2023 2:59 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


@@.> I message you on here cause I cant get on discord until 10/16/2023, hope you understand lol (im using email cause i cant log into github on my school computer).
…
________________________________ From: SharkPool-SP @.> Sent: Wednesday, August 16, 2023 8:35 PM To: SharkPool-SP/SharkPools-Extensions @.> Cc: Surv @.>; Author @.> Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16>) Merged #16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16><#16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16>> into main. — Reply to this email directly, view it on GitHub<#16 (comment)<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#event-10113739368>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/BAGGRIBWMEFSUVEJUHY7CZ3XVVRMHANCNFSM6AAAAAA3SW6T4E. You are receiving this because you authored the thread.Message ID: @.>

okay, thanks for letting me know :)

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1726315936>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRICO2EWQHW6PHVHQEQ3X3HTR7ANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SharkPool-SP commented:
> if possible do you have email i could message you at?
> […](#)
> ________________________________ From: SharkPool-SP ***@***.***> Sent: Tuesday, September 19, 2023 2:59 PM To: SharkPool-SP/SharkPools-Extensions ***@***.***> Cc: Surv ***@***.***>; Author ***@***.***> Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16) @@.> I message you on here cause I cant get on discord until 10/16/2023, hope you understand lol (im using email cause i cant log into github on my school computer). …
> ________________________________ From: SharkPool-SP @.> Sent: Wednesday, August 16, 2023 8:35 PM To: SharkPool-SP/SharkPools-Extensions @.> Cc: Surv @.>; Author @.> Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16<#16>) Merged #16<#16><#16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16>> into main. — Reply to this email directly, view it on GitHub<#16 (comment)<[#16 (comment)](https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#event-10113739368)>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/BAGGRIBWMEFSUVEJUHY7CZ3XVVRMHANCNFSM6AAAAAA3SW6T4E. You are receiving this because you authored the thread.Message ID: @.> okay, thanks for letting me know :) — Reply to this email directly, view it on GitHub<[#16 (comment)](https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1726315936)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRICO2EWQHW6PHVHQEQ3X3HTR7ANCNFSM6AAAAAA3SW6T4E>. You are receiving this because you authored the thread.Message ID: ***@***.***>

I have newgrounds?

SurvExe1Pc commented:
"
his content is blocked by your organization
"
________________________________
From: SharkPool-SP ***@***.***>
Sent: Tuesday, September 19, 2023 11:24 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


if possible do you have email i could message you at?
…
________________________________ From: SharkPool-SP @.> Sent: Tuesday, September 19, 2023 2:59 PM To: SharkPool-SP/SharkPools-Extensions @.> Cc: Surv @.>; Author @.> Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16>) @@.> I message you on here cause I cant get on discord until 10/16/2023, hope you understand lol (im using email cause i cant log into github on my school computer). …
________________________________ From: SharkPool-SP @.> Sent: Wednesday, August 16, 2023 8:35 PM To: SharkPool-SP/SharkPools-Extensions @.> Cc: Surv @.>; Author @.> Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16><#16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16>>) Merged #16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16><#16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16>><#16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16>#16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16>> into main. — Reply to this email directly, view it on GitHub<#16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16> (comment)<#16 (comment)<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#event-10113739368>>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/BAGGRIBWMEFSUVEJUHY7CZ3XVVRMHANCNFSM6AAAAAA3SW6T4E. You are receiving this because you authored the thread.Message ID: @.> okay, thanks for letting me know :) — Reply to this email directly, view it on GitHub<#16 (comment)<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1726315936>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/BAGGRICO2EWQHW6PHVHQEQ3X3HTR7ANCNFSM6AAAAAA3SW6T4E. You are receiving this because you authored the thread.Message ID: @.***>

I have newgrounds?

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1726825883>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIFFGNJO2UW3XXDAZITX3JOYTANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
cant use newgrounds 😭
________________________________
From: a 1 ***@***.***>
Sent: Wednesday, September 20, 2023 8:11 AM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)

"
his content is blocked by your organization
"
________________________________
From: SharkPool-SP ***@***.***>
Sent: Tuesday, September 19, 2023 11:24 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


if possible do you have email i could message you at?
…
________________________________ From: SharkPool-SP @.> Sent: Tuesday, September 19, 2023 2:59 PM To: SharkPool-SP/SharkPools-Extensions @.> Cc: Surv @.>; Author @.> Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16>) @@.> I message you on here cause I cant get on discord until 10/16/2023, hope you understand lol (im using email cause i cant log into github on my school computer). …
________________________________ From: SharkPool-SP @.> Sent: Wednesday, August 16, 2023 8:35 PM To: SharkPool-SP/SharkPools-Extensions @.> Cc: Surv @.>; Author @.> Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16><#16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16>>) Merged #16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16><#16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16>><#16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16>#16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16>> into main. — Reply to this email directly, view it on GitHub<#16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16> (comment)<#16 (comment)<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#event-10113739368>>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/BAGGRIBWMEFSUVEJUHY7CZ3XVVRMHANCNFSM6AAAAAA3SW6T4E. You are receiving this because you authored the thread.Message ID: @.> okay, thanks for letting me know :) — Reply to this email directly, view it on GitHub<#16 (comment)<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1726315936>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/BAGGRICO2EWQHW6PHVHQEQ3X3HTR7ANCNFSM6AAAAAA3SW6T4E. You are receiving this because you authored the thread.Message ID: @.***>

I have newgrounds?

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1726825883>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIFFGNJO2UW3XXDAZITX3JOYTANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SharkPool-SP commented:
> cant use newgrounds 😭
> […](#)

:(

SurvExe1Pc commented:
my school has github, discord and newgrounds blocked
________________________________
From: SharkPool-SP ***@***.***>
Sent: Wednesday, September 20, 2023 2:52 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


cant use newgrounds 😭
…

:(

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1728267126>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIBXBY3PD56APOXOGFTX3M3QNANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
and my other computer is locked till the date i said
________________________________
From: SharkPool-SP ***@***.***>
Sent: Wednesday, September 20, 2023 2:52 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


cant use newgrounds 😭
…

:(

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1728267126>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIBXBY3PD56APOXOGFTX3M3QNANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SharkPool-SP commented:
> and my other computer is locked till the date i said
> […](#)

Crap…. How else can we contact… google docs? (I’m good with dat)

SurvExe1Pc commented:
also fast nice!
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 11:34 AM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


and my other computer is locked till the date i said
…

Crap…. How else can we contact… google docs? (I’m good with dat)

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1729826736>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIBKOZ6AXOFAI3KSR43X3RM6RANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
i just noticed every built in cloud link server is blocked
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 11:34 AM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


and my other computer is locked till the date i said
…

Crap…. How else can we contact… google docs? (I’m good with dat)

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1729826736>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIBKOZ6AXOFAI3KSR43X3RM6RANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
i just got blocked from allorigins.win
________________________________
From: a 1 ***@***.***>
Sent: Thursday, September 21, 2023 12:09 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)

i just noticed every built in cloud link server is blocked
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 11:34 AM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


and my other computer is locked till the date i said
…

Crap…. How else can we contact… google docs? (I’m good with dat)

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1729826736>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIBKOZ6AXOFAI3KSR43X3RM6RANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SharkPool-SP commented:
> i just got blocked from allorigins.win
> […](#)

Why?

SharkPool-SP commented:
> i have no idea cause all google accs are blocked, my school does not like ppl watchin yt so they blocked google accs, i have no ideas tbh, i only have email cause this is my private one
> […](#)

Fuck bruh

SurvExe1Pc commented:
yeah ikr
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 1:02 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


i have no idea cause all google accs are blocked, my school does not like ppl watchin yt so they blocked google accs, i have no ideas tbh, i only have email cause this is my private one
…

Fuck bruh

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1729969508>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIGRTETOPP262QWKG73X3RXKLANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SharkPool-SP commented:
> yeah ikr
> […](#)

Now what

SurvExe1Pc commented:
dunno

________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 1:08 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


yeah ikr
…

Now what

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1729979147>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIB5URN5MUKLOLVJ453X3RYCTANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
im making a dumb chat app using cloudlink, it uses IFrames to display the messages and other ui(like input and send button), and all the code is in the backend(turbowarp), it is a encrypted message platform, you have a random 250 digit code along with a password as the room id (supports multiple rooms), and then all messages you send have a password on them so only people in the same room and using the same password can see the messages you send.
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 3:10 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


dunno
…

…

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730150008>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIFZGNUDRJKGINLOHS3X3SGJXANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 3:10 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


dunno
…

…

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730150008>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIFZGNUDRJKGINLOHS3X3SGJXANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
i currently have NO idea if any of the cloudlink stuff works due to all the servers and python being blocked
________________________________
From: a 1 ***@***.***>
Sent: Thursday, September 21, 2023 4:32 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)

im making a dumb chat app using cloudlink, it uses IFrames to display the messages and other ui(like input and send button), and all the code is in the backend(turbowarp), it is a encrypted message platform, you have a random 250 digit code along with a password as the room id (supports multiple rooms), and then all messages you send have a password on them so only people in the same room and using the same password can see the messages you send.
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 3:10 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


dunno
…

…

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730150008>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIFZGNUDRJKGINLOHS3X3SGJXANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SharkPool-SP commented:
> i currently have NO idea if any of the cloudlink stuff works due to all the servers and python being blocked
> […](#)

prob you're computer. Btw you missed alot

SurvExe1Pc commented:
WHAT DID I MISS 😭
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 4:41 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


i currently have NO idea if any of the cloudlink stuff works due to all the servers and python being blocked
…

prob you're computer. Btw you missed alot

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730269795>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIGFDLWOTGKAR7KWM6LX3SRA5ANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
Im going to attempt to reset my discord password without the 2FA tokens lol
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 3:10 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


dunno
…

…

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730150008>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIFZGNUDRJKGINLOHS3X3SGJXANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
heh i reset the password time to see what i missed
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 4:41 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


i currently have NO idea if any of the cloudlink stuff works due to all the servers and python being blocked
…

prob you're computer. Btw you missed alot

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730269795>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIGFDLWOTGKAR7KWM6LX3SRA5ANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
ofc only if it manages to load
________________________________
From: a 1 ***@***.***>
Sent: Thursday, September 21, 2023 4:48 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)

heh i reset the password time to see what i missed
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 4:41 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


i currently have NO idea if any of the cloudlink stuff works due to all the servers and python being blocked
…

prob you're computer. Btw you missed alot

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730269795>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIGFDLWOTGKAR7KWM6LX3SRA5ANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
BUT STILL TELL ME WHAT I MISSED PLEASE
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 4:41 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


i currently have NO idea if any of the cloudlink stuff works due to all the servers and python being blocked
…

prob you're computer. Btw you missed alot

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730269795>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIGFDLWOTGKAR7KWM6LX3SRA5ANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
WHA IT WONT LOAD
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 4:41 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


i currently have NO idea if any of the cloudlink stuff works due to all the servers and python being blocked
…

prob you're computer. Btw you missed alot

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730269795>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIGFDLWOTGKAR7KWM6LX3SRA5ANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
can you pls DM dragocuven that "bob_iam" on scratch is me, and send this as proof, and that i want him to log onto scratch.
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 4:41 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


i currently have NO idea if any of the cloudlink stuff works due to all the servers and python being blocked
…

prob you're computer. Btw you missed alot

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730269795>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIGFDLWOTGKAR7KWM6LX3SRA5ANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SharkPool-SP commented:
> BUT STILL TELL ME WHAT I MISSED PLEASE
> […](#)

Shovel and nameless is back. (Server wise)

I added and updated a shit ton of new/existing extensions. (Check them out)

I also make sprite parenting

SurvExe1Pc commented:
github is also blocked thats why im doing it over email
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 4:58 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


BUT STILL TELL ME WHAT I MISSED PLEASE
…

Shovel and nameless is back. (Server wise)

I added and updated a shit ton of new/existing extensions. (Check them out)

I also make sprite parenting

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730293797>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIE7VGU2SPCMZ2GX6R3X3STAPANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SharkPool-SP commented:
> github is also blocked thats why im doing it over email
> […](#)

sad, guess youll see it when ur back... trust me, really cool shit

SharkPool-SP commented:
> github is also blocked thats why im doing it over email
> […](#)

I also made a Mic Recording extension.

SurvExe1Pc commented:
PWEAS SEND DM DRAGO FOR ME
________________________________
From: a 1 ***@***.***>
Sent: Thursday, September 21, 2023 4:58 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)

can you pls DM dragocuven that "bob_iam" on scratch is me, and send this as proof, and that i want him to log onto scratch.
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 4:41 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


i currently have NO idea if any of the cloudlink stuff works due to all the servers and python being blocked
…

prob you're computer. Btw you missed alot

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730269795>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIGFDLWOTGKAR7KWM6LX3SRA5ANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
ill use JSdelivr to go on the repo!

________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 5:00 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


github is also blocked thats why im doing it over email
…

sad, guess youll see it when ur back... trust me, really cool shit

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730295302>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIAPGNWFBV6FXBLGHRLX3STFZANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SharkPool-SP commented:
> PWEAS SEND DM DRAGO FOR ME
> […](#)

I did dw

SurvExe1Pc commented:
thank you ima check out the stuff now 😄 (not discord related)
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 5:03 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


PWEAS SEND DM DRAGO FOR ME
…

I did dw

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730298679>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIF5O3YFTJ7ARWL56U3X3STQLANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
is it on github?
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 5:03 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


PWEAS SEND DM DRAGO FOR ME
…

I did dw

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730298679>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIF5O3YFTJ7ARWL56U3X3STQLANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SharkPool-SP commented:
> is it on github?
> […](#)

Ye

SharkPool-SP commented:
> is it on github?
> […](#)

Oh yeah I also got newgrounds audio fetching to work :)))

SurvExe1Pc commented:
Still a legal grey area lol
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 5:13 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


is it on github?
…

Oh yeah I also got newgrounds audio fetching to work :)))

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730309826>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIGZJSC63LFFQN6IMH3X3SUWBANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SurvExe1Pc commented:
😭 i can no longer email a really old friend, his email no longer exists
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 5:13 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


is it on github?
…

Oh yeah I also got newgrounds audio fetching to work :)))

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730309826>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIGZJSC63LFFQN6IMH3X3SUWBANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SharkPool-SP commented:
> Still a legal grey area lol
> […](#)
> ________________________________ From: SharkPool-SP ***@***.***> Sent: Thursday, September 21, 2023 5:13 PM To: SharkPool-SP/SharkPools-Extensions ***@***.***> Cc: Surv ***@***.***>; Author ***@***.***> Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16) is it on github? … Oh yeah I also got newgrounds audio fetching to work :))) — Reply to this email directly, view it on GitHub<[#16 (comment)](https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730309826)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIGZJSC63LFFQN6IMH3X3SUWBANCNFSM6AAAAAA3SW6T4E>. You are receiving this because you authored the thread.Message ID: ***@***.***>

Dont care, it works and I can get more stuff than just the audio ;)

SharkPool-SP commented:
> 😭 i can no longer email a really old friend, his email no longer exists
> […](#)
> ________________________________ From: SharkPool-SP ***@***.***> Sent: Thursday, September 21, 2023 5:13 PM To: SharkPool-SP/SharkPools-Extensions ***@***.***> Cc: Surv ***@***.***>; Author ***@***.***> Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16) is it on github? … Oh yeah I also got newgrounds audio fetching to work :))) — Reply to this email directly, view it on GitHub<[#16 (comment)](https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730309826)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIGZJSC63LFFQN6IMH3X3SUWBANCNFSM6AAAAAA3SW6T4E>. You are receiving this because you authored the thread.Message ID: ***@***.***>

that sucks

SurvExe1Pc commented:
[cid:c5a1f2fe-726a-416b-a817-f75a4cc8f58f] "were" is our friend group.
________________________________
From: SharkPool-SP ***@***.***>
Sent: Thursday, September 21, 2023 11:37 PM
To: SharkPool-SP/SharkPools-Extensions ***@***.***>
Cc: Surv ***@***.***>; Author ***@***.***>
Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16)


😭 i can no longer email a really old friend, his email no longer exists
…
________________________________ From: SharkPool-SP @.> Sent: Thursday, September 21, 2023 5:13 PM To: SharkPool-SP/SharkPools-Extensions @.> Cc: Surv @.>; Author @.> Subject: Re: [SharkPool-SP/SharkPools-Extensions] Fix json (PR #16<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16>) is it on github? … Oh yeah I also got newgrounds audio fetching to work :))) — Reply to this email directly, view it on GitHub<#16 (comment)<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730309826>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/BAGGRIGZJSC63LFFQN6IMH3X3SUWBANCNFSM6AAAAAA3SW6T4E. You are receiving this because you authored the thread.Message ID: @.***>

that sucks

—
Reply to this email directly, view it on GitHub<https://github.com/SharkPool-SP/SharkPools-Extensions/pull/16#issuecomment-1730757177>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAGGRIDXRG6A5LXSFMUFRMTX3UBZHANCNFSM6AAAAAA3SW6T4E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SharkPool-SP commented:
> [cid:c5a1f2fe-726a-416b-a817-f75a4cc8f58f] "were" is our friend group.
> […](#)

What?
